packages/vm/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  extends: "@ethereumjs/eslint-config-defaults",
+  ignorePatterns: ["tests", "scripts", "examples", "karma.conf.js"],
+  rules: {
+    '@typescript-eslint/await-thenable': 'off',
+    '@typescript-eslint/no-floating-promises': 'off',
+    '@typescript-eslint/no-unused-vars': 'off',
+    '@typescript-eslint/no-use-before-define': 'off',
+    '@typescript-eslint/restrict-plus-operands': 'off',
+    'no-async-promise-executor': 'off',
+    'no-invalid-this': 'off',
+    'no-restricted-syntax': 'off',
+    'no-undef': 'off',
+    'no-unused-vars': 'off',
+    'prefer-const': 'off',
+    'sonarjs/no-duplicated-branches': 'off',
+  }
```

Comment:
Ah, so many exceptions for the VM necessary? 😄 

No problem of course on this round. Nevertheless interesting to state and see.

Reply:
yes! there were so many eslint complaints. I decided to add those as exceptions for now, and ship the config sooner, as it's working perfectly. 

We can tackle those in other PRs.